packages/evidence/cli.js

```diff
@@ -79,7 +79,8 @@ prog
   .describe("launch the local evidence development environment")
   .action((args) => {
     populateTemplate()
-    const watcher = runFileWatcher()
+    const watcher = runFileWatcher('./pages/','./.evidence/template/src/pages/')
+    const imgWatcher = runFileWatcher('./static/','./.evidence/template/static/')
```

Comment:
@archiewood my concern here is that we're spinning up two separate chokidar "watcher" processes. Instead of doing this, I'd prefer we modify the one watcher to watch both the pages, and the static directory.